src/Elastic.Apm/Agent.cs

```diff
@@ -36,21 +37,23 @@ public interface IApmAgent
 
 		IPayloadSender PayloadSender { get; }
 
+		Service Service { get; }
```

Comment:
What's the idea behind this? If I get this correctly this is actually unrelated to the logger. I'm asking this, because with this the `Service` will be tied to the agent. That is technically correct now, but may be a problem later. We are discussing to have multiple services in a single agent. See https://github.com/elastic/apm-agent-dotnet/issues/59 and https://github.com/elastic/apm-server/issues/1175